crates/db_schema/src/impls/password_reset_request.rs

```diff
@@ -33,6 +35,18 @@ impl Crud for PasswordResetRequest {
   }
   async fn create(pool: &DbPool, form: &PasswordResetRequestForm) -> Result<Self, Error> {
     let conn = &mut get_conn(pool).await?;
+    // Expire old tokens
+    // Subtract a few seconds in case DB is on separate server and time isn't perfectly synced
+    let expire_time = naive_now() - Duration::seconds(5);
```

Comment:
I really dont think you need to do any of this weird messing with times, its not needed anywhere else in the code.

Reply:
I wasn't happy with that either, I was originally planning to just use the `CURRENT_TIMESTAMP` from the database, but the `Crud` trait doesn't really provide a way to do that. I reworked it a little by using functions outside the `Crud` trait so it's less messy now. It didn't seem quite right to be modifying other rows as part of a single crud operation anyway.